docs/source/pages/tutorials/use_verification_summary_attestation.rst

```diff
@@ -107,25 +109,54 @@ The VSA adheres to the `schema <https://slsa.dev/spec/v1.0/verification_summary>
             check_passed(component_id, "mcn_provenance_expectation_1")
 
-        apply_policy_to("has-hosted-build", component_id) :-
+        apply_policy_to("gdk_provenance_policy", component_id) :-
             is_component(component_id, purl),
             match("^pkg:maven/io.micronaut/micronaut-core@.*$", purl).
 
-This policy makes sure the :ref:`mcn_provenance_expectation_1 <checks>` check, which verifies the content of the provenance file matches :ref:`CUE expectation <pages/using:Verifying provenance expectations in CUE language>`. You can find the template policy files for GDK builds below:
+    This policy makes sure the :ref:`mcn_provenance_expectation_1 <checks>` check, which verifies the content of the provenance file matches :ref:`CUE expectation <pages/using:Verifying provenance expectations in CUE language>`.
+
+    * Policy prelude (``#include "prelude.dl"``): Copies all the pre-written rules and the generated fact import statements into the policy program. All user-written policy files must begin with ``#include "prelude.dl"``.
+
+    * Policy Validation (``Policy``): This rule ensures that the component satisfies the ``mcn_provenance_expectation_1`` check.
+
+    * Applying the Policy (``apply_policy_to``): To apply the ``gcn_provenance_policy``, Macaron first determines if the ``component_id`` is a valid component and if its ``PURL`` conforms to the pattern defined in the ``match`` predicate. If both conditions are met, the policy is applied.
+
+    * The template Datalog policy file can be downloaded from `here <https://github.com/oracle/macaron/tree/main/src/macaron/resources/policies/gdk/policy.dl.template>`_
+
+    Below you can find the template CUE file that has been used by the :ref:`mcn_provenance_expectation_1 <checks>` check at verification time to verify the provenance. It contains place holders for expected values that are populated by the GDK maintainers.
+
+    .. code-block:: javascript
 
-* `Template CUE expectation <https://github.com/oracle/macaron/tree/main/src/macaron/resources/policies/gdk/expectation.cue.template>`_
-* `Template Datalog policy file <https://github.com/oracle/macaron/tree/main/src/macaron/resources/policies/gdk/policy.dl.template>`_
+        {
+            predicate: {
+                attestations: [
+                    {
+                        attestation: {
+                            jobimage: "<IMAGE-ADDRESS>",
+                            projecturl: "https://<REPO_URL>",
+                        },
+                    },
+                ]
+            }
+        }
+
+
+    * ``jobimage: "<IMAGE-ADDRESS>"``: This condition checks that the ``jobimage`` attribute matches a specific pattern. ``<IMAGE-ADDRESS>`` is a placeholder for the actual image name used at build time.
+
+    * ``projecturl: "https://<REPO_URL>"``: This checks that the ``projecturl`` attribute exactly matches the expected Repository URL. ``<REPO_URL>`` is a placeholder for the actual repository URL.
```

Comment:
Do we assume that the repository URL would always start with `https://` ?
Do you think it would be good to have a small example on what the values for `IMAGE-NAME` and `REPO-URL` could look like 🤔. I wouldn't mind keeping it this way either.

Reply:
See commit [7ffe436](https://github.com/oracle/macaron/pull/858/commits/7ffe436657863c3a72133e32884d6c871d90836e).